Add signed package verification to TOC and make more descriptive (#32818)
dotnet/docs: docs/core/tools/nuget-signed-package-verification.md

author: DTivel
ms.date: 11/07/2022
---
# NuGet signed-package verification

> A [signed NuGet package](/nuget/create-packages/sign-a-package) enables a consumer to validate the package's authenticity and integrity. If verification is enabled, .NET verifies signed packages during a package restore operation, which occurs automatically when you build or run your project.

Reviewer: ```suggestion
You can [sign a NuGet package](/nuget/create-packages/sign-a-package) to enable the package consumer to validate the package's authenticity and integrity. If verification is enabled, .NET verifies signed packages during a package restore operation, which occurs automatically when the consumer builds or runs their project.
```

Verification is enabled by default during package restore operations.

NuGet uses the default root store on Windows, which already supports general-purpose code signing and timestamping. .NET SDK fallback certificate bundles aren't used.

## Linux

Verification is disabled by default during package restore operations. To opt in, set the environment variable `DOTNET_NUGET_SIGNATURE_VERIFICATION` to `true`.

NuGet uses .NET SDK fallback certificate bundles by default. You can override the code signing fallback certificate bundle by providing a certificate bundle valid for code signing at the following probe path:

```text
/etc/pki/ca-trust/extracted/pem/objsign-ca-bundle.pem
```

## macOS

> Verification is disabled by default during package restore operations. To opt in, set the environment variable `DOTNET_NUGET_SIGNATURE_VERIFICATION` to `true`. However, it's recommended that you don't enable verification. For more information, see [NuGet/Home#11985](https://github.com/NuGet/Home/issues/11985) and [NuGet/Home#11986](https://github.com/NuGet/Home/issues/11986).

Reviewer: "that you not enable" sounds 

```suggestion
Verification is disabled by default during package restore operations. To opt in, set the environment variable `DOTNET_NUGET_SIGNATURE_VERIFICATION` to `true`. However, we recommend that you don't enable verification. For more information, see [NuGet/Home#11985](https://github.com/NuGet/Home/issues/11985) and [NuGet/Home#11986](https://github.com/NuGet/Home/issues/11986).
```
